app/models/concerns/scholars_archive/to_solr_behavior.rb

```diff
@@ -0,0 +1,25 @@
+module ScholarsArchive
+  module ToSolrBehavior
```

Comment:
I feel like this would be better named relative to what it is doing, `HasSolrNestedGeoLabels`.. what do you think?